base/linalg/symmetric.jl

```diff
@@ -201,6 +201,24 @@ function copy!(dest::Hermitian, src::Hermitian)
     return dest
 end
 
+# fill[stored]!
+function fill!(A::HermOrSym{T}, x) where T
+    xT = convert(T, x)
+    if !iszero(xT)
```

Comment:
I imagine I miss something: A general symmetric matrix can be homogeneously populated with any value, and a general hermitian matrix with any real value?